roles/openshift_setup/tasks/patch_dnsmasq.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Ensure file is present
+  register: _stat_networking_info
+  ansible.builtin.stat:
+    path: /etc/ci/env/networking-info.yml
+
+- name: Load network parameters
+  when: _stat_networking_info.stat.exists
+  block:
+    - name: Slurp network-info.yml
+      register: _patch_dns_network_info_slurp
+      ansible.builtin.slurp:
+        src: "/etc/ci/env/networking-info.yml"
+
+    - name: Decode network-info.yml
+      ansible.builtin.set_fact:
+        _decoded_net_env: >-
+          {{
+            _patch_dns_network_info_slurp['content'] |
+            b64decode | from_yaml
+          }}
+
+- name: Check which dnsmasq config we must edit
+  register: _dnsmasq
+  ansible.builtin.stat:
+    path: '/srv/dnsmasq.conf'
+    get_attributes: false
+    get_checksum: false
+    get_mime: false
+
+
+- name: Shared parameter across blocks
+  vars:
+    _crc_default_net_ip: >-
+      {{
+        _decoded_net_env.crc_ci_bootstrap_networks_out.crc.default.ip | default('192.168.130.11')
```

Comment:
here we need to add a `default(_decoded_net_env.crc_ci_bootstrap_networks_out['crc-0'].default.ip)` since the crc vm in the reproducer case is called `crc-0`